docs/README.md

```diff
@@ -183,7 +185,7 @@ We can quickly create a bootstrapped database and start the Fuel indexer service
 ```bash
 forc index start \
     --embedded-database                         # Setup and start a default database.
-    --fuel-node-host node-beta-3.fuel.network \ # Connect to a Fuel node at this host
+    --fuel-node-host beta-3.fuel.network \ # Connect to a Fuel node at this host
     --fuel-node-port 80                         # and port, and monitor the network.
 ```
```

Comment:
I know I was the one that originally wrote this blurb, but it turns out that copy-pasting without the backslashes on the ends of the comments. Can we change it to
```
forc index start \
    --embedded-database \                        
    --fuel-node-host beta-3.fuel.network \
    --fuel-node-port 80 
```

and then change the preceding line to the following?: 
> "For this example, we'll quickly create a bootstrapped database, start the Fuel indexer service, and monitor the beta-3 testnet network by running the following command:"